ci-operator/config/openshift/machine-api-operator/openshift-machine-api-operator-master.yaml

```diff
@@ -233,6 +237,23 @@ tests:
   steps:
     cluster_profile: openstack-vexxhost
     workflow: openshift-e2e-openstack-ipi
+- as: regression-clusterinfra-aws-ipi-mapi
+  optional: true
+  run_if_changed: ^(go\.mod|go\.sum)$
```

Comment:
This is to run the job only on dependency bumps, and not on normal code changes right?

Reply:
Right, I am adding this job aim to run mapi regression cases when machine-api-operator repo rebases